google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -52,12 +52,26 @@
 import java.util.concurrent.TimeUnit;
 
 /**
- * A Google cloud storage object.
+ * A Google cloud storage object. A {@code Blob} object includes the {@code BlobId} instance, the
+ * set of properties inherited from the {@link BlobInfo} class and the {@code Storage} instance. The
+ * class provides methods to perform operations on the object. Reading a property value does not
+ * issue any RPC calls. The object content is not stored within the {@code Blob} instance,
+ * operations on obtaining the content will issue one or multiple RPC calls, depending on the
```

Comment:
Operations that access the content issue...

Reply:
done.  But _access the content issue_ might sound confusing.